### ISSUE TITLE
In App USB/Bluetooth Hotspot Toggles Not working 

### ISSUE THREAD

**Issue by C00LDroid** [Mygod/VPNHotspot#14]
In App USB/Bluetooth Hotspot Toggles Not working
Getting error Neither user nor current process has permission....(check ss) while usb tethering and Android system has failed to start Tethering for Bluetooth tethering though WiFi  #tethering working fine. Using Android 8.0 Mi A1 Device.
![screenshot_20180424-182041](https://user-images.githubusercontent.com/38692811/39188330-2ab0658c-47ed-11e8-98d3-cd6828173747.png)
![screenshot_20180424-182035](https://user-images.githubusercontent.com/38692811/39188348-30fc235e-47ed-11e8-8aac-49a9d98de447.png)

**Comment by Mygod** [comment]
It's expected if you:

1. For BT, didn't turn on Bluetooth;
2. For USB, didn't install this app as a system app (see readme).

This is a just for fun feature. You could use system tethering activity to switch stuff properly. :)

**Comment by Mygod** [comment]
There's no need for that since this app makes no change to `/system`. Did you lose system update by using this app?

**Comment by C00LDroid** [comment]
Never installed it as a system app so can't say so surely however touching /system partition any way results in losing system updates

**Comment by Mygod** [comment]
Oh I see. I think there's a Magisk module called App systemizer that does exactly that. I tried that once while developing this app but it ended up bootlooping my device (maybe it doesn't support Magisk 16 yet). :stuck_out_tongue: So I just moved apk to `/system/priv-app` manually and it worked.

You really don't need to install this app as system app if you can toggle system tetherings from your system settings (you can press "Manage..." to go there).

**Comment by C00LDroid** [comment]
Yeah this is not Must have feature[magisk module] still consider this in further development, Thanks.. Loved your app..😉

**Comment by Mygod** [comment]
For future references, this app has been tested with [App systemizer](https://forum.xda-developers.com/apps/magisk/module-terminal-app-systemizer-ui-t3585851) v14.1.

**Comment by Mygod** [comment]
This issue will be resolved in upcoming update v2.10.2.
